app/assets/stylesheets/application.css

```diff
@@ -11,3 +11,7 @@
  * It is generally better to create a new file per style scope.
  *
  */
+
+table {
+	margin: 10px
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 2 spaces   indentation